src/core/server/opensearch/opensearch_config.ts

```diff
@@ -80,6 +80,10 @@ export const configSchema = schema.object({
   customHeaders: schema.recordOf(schema.string(), schema.string(), { defaultValue: {} }),
   shardTimeout: schema.duration({ defaultValue: '30s' }),
   requestTimeout: schema.duration({ defaultValue: '30s' }),
+  memoryCircuitBreaker: schema.object({
```

Comment:
nit: same as above i don't think we need to split the Timeouts. Easier to see if they are batched together.